pendulum/time.py

```diff
@@ -19,7 +19,7 @@
 from pendulum.mixins.default import FormattableMixin
 
 if TYPE_CHECKING:
-    from typing import Literal
+    from pendulum.utils._compat import Literal
```

Comment:
This should be changed to `from typing_extensions import Literal` so that pyright is supported